src/components/video-player/video-player.js

```diff
@@ -113,7 +110,7 @@ class VideoPlayer extends Component {
   render(): React$Element<any> {
     const {videoStyles, targetId} = this.props;
 
-    return <div id={`${targetId}-video`} className={style.videoPlayer} style={videoStyles} ref={this._setRef} />;
+    return <div id={`${targetId}-video`} className={style.videoPlayer} style={videoStyles} ref={ref => (ref ? this._setRef(ref) : undefined)} />;
```

Comment:
better not to use anonymous function for ref, can this be handled inside the setRef
also, why do we have logic in setRef? seems problematic. @yairans FYI
Can we move this to one of the lifecycle methods? if so lets handle it in another ticket.